training/import-data.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Importing Data to Your Cluster
+summary: Learn how to import data from pg_dump and CSV files into your CockroachDB cluster.
+toc: false
+sidebar_data: sidebar-data-training.json
+---
+
+<div id="toc"></div>
+
+## Presentation
+
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vRg_445dWS0Ag1Ta3jdMdyfyOIvpP72U0W3XklF8ScJmUlLkdezZUy7JK1jca3A5fWoZiEpq8iu_OMd/embed?start=false&loop=false" frameborder="0" width="756" height="454" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
+
+#### URL for comments
+
+[Importing Data to Your Cluster](https://docs.google.com/presentation/d/13yXxi-7AktFMzq6UlteD7CejTdQE-HLWKXKyUqeebgM/)
+
+## Lab
+
+In this lab, we'll download a dump from PostgreSQL (generated with `pg_dump`), clean it, and then import it into CockroachDB.
```

Comment:
Yeah, it's overly complex. If we had a little more breathing room, I think it would be great, but my fear is that it introduces more complexities and more chances for the class to get hung up. If we find that we _do_ have time, I think you're right that it's what we should be doing.